src/test/regress/operations_schedule

```diff
@@ -2,11 +2,10 @@ test: multi_test_helpers multi_test_helpers_superuser
 test: multi_cluster_management
 test: multi_test_catalog_views
 test: shard_rebalancer_unit
-test: turn_mx_off
```

Comment:
I think this test might be failing on enterprise, I once tried that and failed. Can you please check that as well?